graf2d/graf/src/TCutG.cxx

```diff
@@ -370,27 +370,18 @@ Double_t TCutG::IntegralHist(TH2 *h, Option_t *option) const
 
 void TCutG::SavePrimitive(std::ostream &out, Option_t *option /*= ""*/)
 {
-   char quote = '"';
-   out<<"   "<<std::endl;
-   if (gROOT->ClassSaved(TCutG::Class())) {
-      out<<"   ";
-   } else {
-      out<<"   TCutG *";
-   }
-   out<<"cutg = new TCutG("<<quote<<GetName()<<quote<<","<<fNpoints<<");"<<std::endl;
-   out<<"   cutg->SetVarX("<<quote<<GetVarX()<<quote<<");"<<std::endl;
-   out<<"   cutg->SetVarY("<<quote<<GetVarY()<<quote<<");"<<std::endl;
-   out<<"   cutg->SetTitle("<<quote<<GetTitle()<<quote<<");"<<std::endl;
+   SavePrimitiveConstructor(out, Class(), "cutg", TString::Format("\"%s\", %d", GetName(), fNpoints));
+   out << "   cutg->SetVarX(\"" << GetVarX() << "\");" << std::endl;
+   out << "   cutg->SetVarY(\"" << GetVarY() << "\");" << std::endl;
+   out << "   cutg->SetTitle(\"" << TString(GetTitle()).ReplaceSpecialCppChars() << "\");" << std::endl;
```

Comment:
As a side note, it seems that calling `ReplaceSpecialCppChars` (i.e. escaping quote characters) would be necessary to use every single time a string is being written in the generated source code.

Reply:
e.g the draw option on (new) line 384

Reply:
In general yes - all kind of text or draw options should be handled like that.
And I try to do this in my PR.

For the moment I do not touch object names - normally characters like `"` or `\` not typical there